kustomize/base/am-cdk/testdeployment.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
I think it is better delete this and make a note of it on confluence, under here: https://wikis.forgerock.org/confluence/display/SBAT/Tips+and+Tricks

Link to the article:
https://cloud.google.com/kubernetes-engine/docs/tutorials/workload-identity-secrets#verify

All we need to do is kubectl apply the manifest, so doesn't need to be stored in the forgeops repo in my view.

Reply:
That's been added now to the confluence wiki